src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers} from "redux";
+import {filterReducer}    from "./filter";
+import {booksReducer}     from "./books";
+
+export default combineReducers({
```

Comment:
In relation to my comment on creating a default on `booksReducer` and `filterReducer` would have been improved your this index.js like that

```javascript
import { combineReducers} from "redux";
import filter from "./filter";
import books from "./books";

export default combineReducers({
  filter,
  books
})
```